EL-2517-LabelsSpark

chris-brennan opened the pull request:
Added an example of the spark chart background and bar color to the spark chart table.  Added a table to show an example of the label classes that are available.  Updated the label examples section to use example text rather than the name of the label class, since this is now shown in the label table.

donna-marijne commented on docs/app/pages/css/sections/labels/labels/labels.component.html:
```diff
@@ -1,58 +1,126 @@
+                <th>General Labels</th>
+                <th></th>
+                <th>Outline Labels</th>
+                <th></th>
```
I think this would be better as two tables in column layout. If you use the col-* classes, you can get them to appear vertically for smaller screens.

See how it looks like this:

````
<div class="row">
  <div class="col-md-6 col-sm-12">
    <!-- General labels -->
  </div>
  <div class="col-md-6 col-sm-12">
    <!-- Outline labels -->
  </div>
</div>
````

See how that looks, it might be good with a small gap between the columns too.

donna-marijne commented on docs/app/pages/css/sections/labels/labels/labels.component.html:
```diff
@@ -1,58 +1,126 @@
-<span class="label label-critical">Label Critical</span>
+<span class="label label-ok m-r-sm">Completed</span>
+<span class="label label-warning m-r-sm">Cancelled</span>
+<span class="label label-critical">Failed</span>
```
The example code and codepen are not up to date with this change - see the snippets subdirectory.

chris-brennan commented:
Updated angular 4 section.  Moved spark chart color reference above example code and applied consistent wording with labels section.  Removed extraneous columns from spark chart table (i.e. type, description and optional) and applied `col-md-6`.  Split labels color reference table into two separate tables.  Updated snippets and code pen.

donna-marijne commented on docs/app/pages/charts/sections/spark-charts/spark-chart-ng1/spark-chart-ng1.component.ts:
```diff
@@ -6,7 +6,7 @@ import { ICodePen } from '../../../../../interfaces/ICodePen';
     selector: 'uxd-charts-spark-chart-ng1',
     templateUrl: './spark-chart-ng1.component.html',
-    styleUrls: ['./spark-chart-ng1.component.less'],
+    styleUrls: ['./../spark-charts.less'],
```
I don't think we should do this. Add common documentation styles to the docs/styles.less file.

donna-marijne commented on docs/app/pages/charts/sections/spark-charts/spark-chart-ng1/spark-chart-ng1.component.html:
```diff
@@ -128,6 +128,95 @@
+                    <tr>
+                        <td class="attribute">spark-chart1</td>
+                        <td class="color-example-cell">
+                            <div class="spark spark-chart1 bg-color-example">
```
I didn't notice this before - this should be an actual spark directive.

donna-marijne commented on docs/app/pages/charts/sections/spark-charts/spark-charts/spark-charts.component.html:
```diff
@@ -115,6 +118,95 @@
     </table>
 </div>
 
+<p>UX Aspects offers a wide range of chart colors. Set the color of the chart by using one of the following classes:</p>
```
Since the angular 4 component allows arbitrary colors via barColor and trackColor, this description needs to be a bit different. Also the color service for Angular 4 has not yet been documented. I think doing this will take quite a bit more effort, so for now can you just revert the changes in the Angular 4 component? Sorry for the backtrack.

chris-brennan commented:
Wording of documentation text updated, spark directive used instead of CSS styles, common documentation styles moved to docs/styles.css, Angular 4 spark chart documentation reverted.
